### PR TITLE
prepare lib for rollup by consumer

### PR DESCRIPTION
**Pull request by pavadeli**
Creates several bundle types with appropriate source maps and type definitions. Also a small change in API to allow for more efficient rollup by consumers.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import builtins from 'rollup-plugin-node-builtins';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+import pkg from './package.json';
+
+export default [
+    // browser-friendly UMD build
```
Comment (WouterSpaak) [inline rollup.config.js]: As we're supporting `<script src="...">` imports, shouldn't we also bundle a minified version of the UMD file?

File: package.json [hunk 2]
```diff
@@ -2,17 +2,21 @@
     "name": "@politie/informant",
     "version": "1.0.0-beta.0",
     "description": "Informant is a simple and fast logging library for Node.js and browser JavaScript applications.",
-    "main": "dist/index.js",
+    "main": "dist/informant.cjs.js",
```
Comment (WouterSpaak) [inline package.json]: As we're also bundling all transpiled source files, I think having the rollup bundles in a folder `dist/bundles/<bundle>.<type>.js` might be a bit clearer.

Comment (pavadeli) [reply package.json]: The bundles are the only JS files in the `dist` directory, so an additional directory seems like overkill to me.

File: rollup.config.js (hunk 1 above)
Comment (pavadeli) [reply rollup.config.js]: We could, but the only reason I added this build is for StackBlitz support, I don't think people are going to use this package without a bundler.

File: package.json (hunk 2 above)
Comment (pavadeli) [reply package.json]: Note that only the three bundles are packed inside the NPM package.

Comment (WouterSpaak) [reply package.json]: Fair enough. Didn't run `npm pack` yet. LGTM.

File: rollup.config.js (hunk 1 above)
Comment (WouterSpaak) [reply rollup.config.js]: All right. Let's keep it like this until someone wants a minified file.

**Pull request merged**